### ISSUE TITLE
Not listing from trakt

### ISSUE THREAD

**Issue by Jezze83** [scakemyer/plugin.video.quasar#565]
I have raspberry pi 3. I upgraded openelec to jarvis version and i have quasar v0.9.36. Works fine if i dont select from trakt and my trakt account is authorized. When i choose trakt and trending openelec says something: get https://api-v2launch.trakt.tv/movies/tr... Dont know whole url because openelec wont show it and it doesnt show logs either.

I just tested authorize again quasar to trakt and now it gives navigation error: httperror() when pressing Authorize Quasar.

```
15:26:06 125.937874 T:1573909408  NOTICE: Previous line repeats 1 times.
15:26:06 125.937920 T:1573909408  NOTICE: [plugin.video.quasar] [31m2016/11/17 13:26:06 [Recovery] panic recovered:
15:26:06 125.938194 T:1573909408  NOTICE: [plugin.video.quasar] GET /shows/ HTTP/1.1
15:26:06 125.938469 T:1573909408  NOTICE: [plugin.video.quasar] Host: localhost:65251
15:26:06 125.938736 T:1573909408  NOTICE: [plugin.video.quasar] Connection: close
15:26:06 125.939011 T:1573909408  NOTICE: [plugin.video.quasar] Accept-Encoding: identity
15:26:06 125.939278 T:1573909408  NOTICE: [plugin.video.quasar] Connection: close
15:26:06 125.939552 T:1573909408  NOTICE: [plugin.video.quasar] User-Agent: Python-urllib/2.7
15:26:06 125.939796 T:1573909408  NOTICE: [plugin.video.quasar]
15:26:06 125.940536 T:1573909408  NOTICE: Previous line repeats 1 times.
15:26:06 125.940582 T:1573909408  NOTICE: [plugin.video.quasar] open /storage/.kodi/userdata/addon_data/plugin.video.quasar/cache/io.scakemyer.quasar.page.cache:e168e2e86c76a6f03aa5456dc70bdd7d66cbecfe: no such file or directory
15:26:06 125.940895 T:1573909408  NOTICE: [plugin.video.quasar] /usr/local/go/src/runtime/panic.go:458 (0x1018b8)
15:26:06 125.941208 T:1573909408  NOTICE: [plugin.video.quasar] /go/src/github.com/gin-gonic/gin/context.go:420 (0x33a030)
15:26:06 125.941528 T:1573909408  NOTICE: [plugin.video.quasar] /go/src/github.com/scakemyer/quasar/api/shows.go:42 (0x209be0)
15:26:06 125.941849 T:1573909408  NOTICE: [plugin.video.quasar] /go/src/github.com/gin-gonic/gin/context.go:97 (0x338658)
15:26:06 125.942665 T:1573909408  NOTICE: [plugin.video.quasar] /go/src/github.com/scakemyer/quasar/cache/cache.go:114 (0x44890c)
15:26:07 126.102829 T:1573909408  NOTICE: [plugin.video.quasar] /go/src/github.com/gin-gonic/gin/context.go:97 (0x338658)
15:26:07 126.103943 T:1573909408  NOTICE: [plugin.video.quasar] /go/src/github.com/gin-gonic/gin/recovery.go:45 (0x347700)
15:26:07 126.104576 T:1573909408  NOTICE: [plugin.video.quasar] /go/src/github.com/gin-gonic/gin/context.go:97 (0x338658)
15:26:07 126.104950 T:1573909408  NOTICE: [plugin.video.quasar] /go/src/github.com/gin-gonic/gin/logger.go:63 (0x346b70)
15:26:07 126.105270 T:1573909408  NOTICE: [plugin.video.quasar] /go/src/github.com/gin-gonic/gin/context.go:97 (0x338658)
15:26:07 126.105583 T:1573909408  NOTICE: [plugin.video.quasar] /go/src/github.com/gin-gonic/gin/gin.go:284 (0x33e138)
15:26:07 126.105904 T:1573909408  NOTICE: [plugin.video.quasar] /go/src/github.com/gin-gonic/gin/gin.go:265 (0x33de44)
15:26:07 126.106209 T:1573909408  NOTICE: [plugin.video.quasar] /usr/local/go/src/net/http/server.go:2022 (0x18fd38)
15:26:07 126.106522 T:1573909408  NOTICE: [plugin.video.quasar] /usr/local/go/src/net/http/server.go:2202 (0x1906bc)
15:26:07 126.106865 T:1573909408  NOTICE: [plugin.video.quasar] /usr/local/go/src/net/http/server.go:1579 (0x18d854)
15:26:07 126.107178 T:1573909408  NOTICE: [plugin.video.quasar] /usr/local/go/src/runtime/asm_arm.s:998 (0x132918)
15:26:07 126.107430 T:1573909408  NOTICE: [plugin.video.quasar] [0m
15:26:07 126.108627 T:1573909408  NOTICE: [plugin.video.quasar] [GIN] 2016/11/17 - 13:26:06 |[97;41m   0 [0m|  136.100639ms | 127.0.0.1 |[97;44m  [0m GET     /shows/
15:26:09 128.251373 T:1703793568  NOTICE: [plugin.video.quasar] Requesting http://localhost:65251/shows/trakt/ from ['plugin://plugin.video.quasar/shows/trakt/', '18', '']
15:26:09 128.403168 T:1573909408  NOTICE: [plugin.video.quasar]
15:26:09 128.404419 T:1573909408  NOTICE: Previous line repeats 1 times.
15:26:09 128.404495 T:1573909408  NOTICE: [plugin.video.quasar] [31m2016/11/17 13:26:09 [Recovery] panic recovered:
15:26:09 128.404800 T:1573909408  NOTICE: [plugin.video.quasar] GET /shows/trakt/ HTTP/1.1
15:26:09 128.405075 T:1573909408  NOTICE: [plugin.video.quasar] Host: localhost:65251
15:26:09 128.405350 T:1573909408  NOTICE: [plugin.video.quasar] Connection: close
15:26:09 128.405640 T:1573909408  NOTICE: [plugin.video.quasar] Accept-Encoding: identity
15:26:09 128.405899 T:1573909408  NOTICE: [plugin.video.quasar] Connection: close
15:26:09 128.406204 T:1573909408  NOTICE: [plugin.video.quasar] User-Agent: Python-urllib/2.7
15:26:09 128.406464 T:1573909408  NOTICE: [plugin.video.quasar]
15:26:09 128.407181 T:1573909408  NOTICE: Previous line repeats 1 times.
15:26:09 128.407211 T:1573909408  NOTICE: [plugin.video.quasar] open /storage/.kodi/userdata/addon_data/plugin.video.quasar/cache/io.scakemyer.quasar.page.cache:d5d82a157d0ee47ae9530dda8337f6df66a3d4ea: no such file or directory
15:26:09 128.407532 T:1573909408  NOTICE: [plugin.video.quasar] /usr/local/go/src/runtime/panic.go:458 (0x1018b8)
15:26:09 128.407852 T:1573909408  NOTICE: [plugin.video.quasar] /go/src/github.com/gin-gonic/gin/context.go:420 (0x33a030)
15:26:09 128.408173 T:1573909408  NOTICE: [plugin.video.quasar] /go/src/github.com/scakemyer/quasar/api/shows.go:69 (0x20ad04)
15:26:09 128.408478 T:1573909408  NOTICE: [plugin.video.quasar] /go/src/github.com/gin-gonic/gin/context.go:97 (0x338658)
15:26:09 128.408813 T:1573909408  NOTICE: [plugin.video.quasar] /go/src/github.com/scakemyer/quasar/cache/cache.go:114 (0x44890c)
15:26:09 128.409134 T:1573909408  NOTICE: [plugin.video.quasar] /go/src/github.com/gin-gonic/gin/context.go:97 (0x338658)
15:26:09 128.409454 T:1573909408  NOTICE: [plugin.video.quasar] /go/src/github.com/gin-gonic/gin/recovery.go:45 (0x347700)
15:26:09 128.409760 T:1573909408  NOTICE: [plugin.video.quasar] /go/src/github.com/gin-gonic/gin/context.go:97 (0x338658)
15:26:09 128.410095 T:1573909408  NOTICE: [plugin.video.quasar] /go/src/github.com/gin-gonic/gin/logger.go:63 (0x346b70)
15:26:09 128.410431 T:1573909408  NOTICE: [plugin.video.quasar] /go/src/github.com/gin-gonic/gin/context.go:97 (0x338658)
15:26:09 128.410736 T:1573909408  NOTICE: [plugin.video.quasar] /go/src/github.com/gin-gonic/gin/gin.go:284 (0x33e138)
15:26:09 128.411057 T:1573909408  NOTICE: [plugin.video.quasar] /go/src/github.com/gin-gonic/gin/gin.go:265 (0x33de44)
15:26:09 128.411362 T:1573909408  NOTICE: [plugin.video.quasar] /usr/local/go/src/net/http/server.go:2022 (0x18fd38)
15:26:09 128.411682 T:1573909408  NOTICE: [plugin.video.quasar] /usr/local/go/src/net/http/server.go:2202 (0x1906bc)
15:26:09 128.411987 T:1573909408  NOTICE: [plugin.video.quasar] /usr/local/go/src/net/http/server.go:1579 (0x18d854)
15:26:09 128.412292 T:1573909408  NOTICE: [plugin.video.quasar] /usr/local/go/src/runtime/asm_arm.s:998 (0x132918)
15:26:09 128.412552 T:1573909408  NOTICE: [plugin.video.quasar] [0m
15:26:09 128.412933 T:1573909408  NOTICE: [plugin.video.quasar] [GIN] 2016/11/17 - 13:26:09 |[97;41m   0 [0m|    1.316302ms | 127.0.0.1 |[97;44m  [0m GET     /shows/trakt/
15:26:11 130.213013 T:1703793568  NOTICE: [plugin.video.quasar] Requesting http://localhost:65251/shows/trakt/popular from ['plugin://plugin.video.quasar/shows/trakt/popular', '19', '']
15:26:11 130.357773 T:1963933696   ERROR: container.setviewmode called with invalid number of parameters (should be: 1, is 0)
15:26:11 130.517120 T:1573909408  NOTICE: [plugin.video.quasar]
15:26:11 130.520447 T:1573909408  NOTICE: Previous line repeats 1 times.
15:26:11 130.520706 T:1573909408  NOTICE: [plugin.video.quasar] [31m2016/11/17 13:26:11 [Recovery] panic recovered:
15:26:11 130.522568 T:1573909408  NOTICE: [plugin.video.quasar] GET /shows/trakt/popular HTTP/1.1
15:26:11 130.523895 T:1573909408  NOTICE: [plugin.video.quasar] Host: localhost:65251
15:26:11 130.525360 T:1573909408  NOTICE: [plugin.video.quasar] Connection: close
15:26:11 130.526428 T:1573909408  NOTICE: [plugin.video.quasar] Accept-Encoding: identity
15:26:11 130.527084 T:1573909408  NOTICE: [plugin.video.quasar] Connection: close
15:26:11 130.527664 T:1573909408  NOTICE: [plugin.video.quasar] User-Agent: Python-urllib/2.7
15:26:11 130.528229 T:1573909408  NOTICE: [plugin.video.quasar]
15:26:11 130.529572 T:1573909408  NOTICE: Previous line repeats 1 times.
15:26:11 130.529648 T:1573909408  NOTICE: [plugin.video.quasar] open /storage/.kodi/userdata/addon_data/plugin.video.quasar/cache/io.scakemyer.quasar.page.cache:d7d15c1bc67f9c5943667a869500a7cd829ed7fe: no such file or directory
15:26:11 130.530319 T:1573909408  NOTICE: [plugin.video.quasar] /usr/local/go/src/runtime/panic.go:458 (0x1018b8)
15:26:11 130.530960 T:1573909408  NOTICE: [plugin.video.quasar] /go/src/github.com/gin-gonic/gin/context.go:420 (0x33a030)
15:26:11 130.531601 T:1573909408  NOTICE: [plugin.video.quasar] /go/src/github.com/scakemyer/quasar/api/trakt.go:459 (0x21b204)
15:26:11 130.532303 T:1573909408  NOTICE: [plugin.video.quasar] /go/src/github.com/scakemyer/quasar/api/trakt.go:469 (0x21b408)
15:26:11 130.532928 T:1573909408  NOTICE: [plugin.video.quasar] /go/src/github.com/gin-gonic/gin/context.go:97 (0x338658)
15:26:11 130.533661 T:1573909408  NOTICE: [plugin.video.quasar] /go/src/github.com/scakemyer/quasar/cache/cache.go:114 (0x44890c)
15:26:11 130.534302 T:1573909408  NOTICE: [plugin.video.quasar] /go/src/github.com/gin-gonic/gin/context.go:97 (0x338658)
15:26:11 130.534943 T:1573909408  NOTICE: [plugin.video.quasar] /go/src/github.com/gin-gonic/gin/recovery.go:45 (0x347700)
15:26:11 130.535583 T:1573909408  NOTICE: [plugin.video.quasar] /go/src/github.com/gin-gonic/gin/context.go:97 (0x338658)
15:26:11 130.536209 T:1573909408  NOTICE: [plugin.video.quasar] /go/src/github.com/gin-gonic/gin/logger.go:63 (0x346b70)
15:26:11 130.536865 T:1573909408  NOTICE: [plugin.video.quasar] /go/src/github.com/gin-gonic/gin/context.go:97 (0x338658)
15:26:11 130.537506 T:1573909408  NOTICE: [plugin.video.quasar] /go/src/github.com/gin-gonic/gin/gin.go:284 (0x33e138)
15:26:11 130.538132 T:1573909408  NOTICE: [plugin.video.quasar] /go/src/github.com/gin-gonic/gin/gin.go:265 (0x33de44)
15:26:11 130.538788 T:1573909408  NOTICE: [plugin.video.quasar] /usr/local/go/src/net/http/server.go:2022 (0x18fd38)
15:26:11 130.539398 T:1573909408  NOTICE: [plugin.video.quasar] /usr/local/go/src/net/http/server.go:2202 (0x1906bc)
15:26:11 130.540039 T:1573909408  NOTICE: [plugin.video.quasar] /usr/local/go/src/net/http/server.go:1579 (0x18d854)
15:26:11 130.540665 T:1573909408  NOTICE: [plugin.video.quasar] /usr/local/go/src/runtime/asm_arm.s:998 (0x132918)
15:26:11 130.541168 T:1573909408  NOTICE: [plugin.video.quasar] [0m
15:26:11 130.541946 T:1573909408  NOTICE: [plugin.video.quasar] [GIN] 2016/11/17 - 13:26:11 |[97;41m   0 [0m|   75.925685ms | 127.0.0.1 |[97;44m  [0m GET     /shows/trakt/popular
15:26:17 136.261307 T:1788867488  NOTICE: [plugin.video.quasar] Requesting http://localhost:65251/shows/trakt/ from ['plugin://plugin.video.quasar/shows/trakt/', '20', '']
15:26:17 136.268539 T:1573909408  NOTICE: [plugin.video.quasar]
15:26:17 136.415939 T:1573909408  NOTICE: Previous line repeats 1 times.
15:26:17 136.416046 T:1573909408  NOTICE: [plugin.video.quasar] [31m2016/11/17 13:26:17 [Recovery] panic recovered:
15:26:17 136.416397 T:1573909408  NOTICE: [plugin.video.quasar] GET /shows/trakt/ HTTP/1.1
15:26:17 136.416718 T:1573909408  NOTICE: [plugin.video.quasar] Host: localhost:65251
15:26:17 136.417007 T:1573909408  NOTICE: [plugin.video.quasar] Connection: close
15:26:17 136.417282 T:1573909408  NOTICE: [plugin.video.quasar] Accept-Encoding: identity
15:26:17 136.417603 T:1573909408  NOTICE: [plugin.video.quasar] Connection: close
15:26:17 136.417877 T:1573909408  NOTICE: [plugin.video.quasar] User-Agent: Python-urllib/2.7
15:26:17 136.418121 T:1573909408  NOTICE: [plugin.video.quasar]
15:26:17 136.418793 T:1573909408  NOTICE: Previous line repeats 1 times.
15:26:17 136.418823 T:1573909408  NOTICE: [plugin.video.quasar] open /storage/.kodi/userdata/addon_data/plugin.video.quasar/cache/io.scakemyer.quasar.page.cache:d5d82a157d0ee47ae9530dda8337f6df66a3d4ea: no such file or directory
15:26:17 136.419144 T:1573909408  NOTICE: [plugin.video.quasar] /usr/local/go/src/runtime/panic.go:458 (0x1018b8)
15:26:17 136.419464 T:1573909408  NOTICE: [plugin.video.quasar] /go/src/github.com/gin-gonic/gin/context.go:420 (0x33a030)
15:26:17 136.419785 T:1573909408  NOTICE: [plugin.video.quasar] /go/src/github.com/scakemyer/quasar/api/shows.go:69 (0x20ad04)
15:26:17 136.420151 T:1573909408  NOTICE: [plugin.video.quasar] /go/src/github.com/gin-gonic/gin/context.go:97 (0x338658)
15:26:17 136.420486 T:1573909408  NOTICE: [plugin.video.quasar] /go/src/github.com/scakemyer/quasar/cache/cache.go:114 (0x44890c)
15:26:17 136.420807 T:1573909408  NOTICE: [plugin.video.quasar] /go/src/github.com/gin-gonic/gin/context.go:97 (0x338658)
15:26:17 136.421127 T:1573909408  NOTICE: [plugin.video.quasar] /go/src/github.com/gin-gonic/gin/recovery.go:45 (0x347700)
15:26:17 136.421432 T:1573909408  NOTICE: [plugin.video.quasar] /go/src/github.com/gin-gonic/gin/context.go:97 (0x338658)
15:26:17 136.421738 T:1573909408  NOTICE: [plugin.video.quasar] /go/src/github.com/gin-gonic/gin/logger.go:63 (0x346b70)
15:26:17 136.422058 T:1573909408  NOTICE: [plugin.video.quasar] /go/src/github.com/gin-gonic/gin/context.go:97 (0x338658)
15:26:17 136.422363 T:1573909408  NOTICE: [plugin.video.quasar] /go/src/github.com/gin-gonic/gin/gin.go:284 (0x33e138)
15:26:17 136.422668 T:1573909408  NOTICE: [plugin.video.quasar] /go/src/github.com/gin-gonic/gin/gin.go:265 (0x33de44)
15:26:17 136.422989 T:1573909408  NOTICE: [plugin.video.quasar] /usr/local/go/src/net/http/server.go:2022 (0x18fd38)
15:26:17 136.423294 T:1573909408  NOTICE: [plugin.video.quasar] /usr/local/go/src/net/http/server.go:2202 (0x1906bc)
15:26:17 136.423645 T:1573909408  NOTICE: [plugin.video.quasar] /usr/local/go/src/net/http/server.go:1579 (0x18d854)
15:26:17 136.423996 T:1573909408  NOTICE: [plugin.video.quasar] /usr/local/go/src/runtime/asm_arm.s:998 (0x132918)
15:26:17 136.424286 T:1573909408  NOTICE: [plugin.video.quasar] [0m
15:26:17 136.424713 T:1573909408  NOTICE: [plugin.video.quasar] [GIN] 2016/11/17 - 13:26:17 |[97;41m   0 [0m|    1.322657ms | 127.0.0.1 |[97;44m  [0m GET     /shows/trakt/
```

**Comment by scakemyer** [comment]
Looks like your cache got corrupted, try clearing it from the advanced tab in the add-on's settings.

```
[plugin.video.quasar] open /storage/.kodi/userdata/addon_data/plugin.video.quasar/cache/io.scakemyer.quasar.page.cache:e168e2e86c76a6f03aa5456dc70bdd7d66cbecfe: no such file or directory
```

**Comment by Jezze83** [comment]
I teated it but not help, i get httperror when i press authorize.

**Comment by scakemyer** [comment]
Make sure remote control options are enabled and restart Kodi.

**Comment by Jezze83** [comment]
Both are enabled.

**Comment by Jezze83** [comment]
There is log when i press authorize quasar from settings, i deleted all providers and quasar plugin. Also deleted manually after that folders from addon data.

18:47:08  60.828876 T:1399845792  NOTICE: [plugin.video.quasar] INFO  repository   ▶ fetchChangelog   [0mFetching add-on changelog...
18:47:08  61.138680 T:1399845792  NOTICE: [plugin.video.quasar] [GIN] 2016/12/02 - 16:47:08 |[97;42m 200 [0m|  1.046661614s | 127.0.0.1 |[97;44m  [0m GET     /repository/scakemyer/plugin.video.quasar/addons.xml.md5
18:47:08  61.139252 T:1399845792  NOTICE: [plugin.video.quasar] Error #01: Unable to retrieve the remote's addon.xml file.
18:47:08  61.192242 T:1399845792  NOTICE: [plugin.video.quasar] INFO  repository   ▶ GetAddonFiles    [0mRequest for: addons.xml
18:47:09  61.494465 T:1399845792  NOTICE: [plugin.video.quasar] [33mWARN  repository   ▶ getLastRelease   [0mUnable to find a last tag, using master.
18:47:09  62.193558 T:1698689952  NOTICE: Previous line repeats 1 times.
18:47:09  62.193687 T:1698689952   ERROR: CCurlFile::Open failed with code 404 for http://localhost:65251/repository/scakemyer/plugin.video.quasar/addons.xml
18:47:09  62.193775 T:1698689952   ERROR: CRepositoryUpdateJob[repository.quasar] failed to read or parse directory 'http://localhost:65251/repository/scakemyer/plugin.video.quasar/addons.xml'
18:47:09  62.193832 T:1399845792  NOTICE: [plugin.video.quasar] [GIN] 2016/12/02 - 16:47:09 |[97;42m 200 [0m|  1.001319527s | 127.0.0.1 |[97;44m  [0m GET     /repository/scakemyer/plugin.video.quasar/addons.xml
18:47:09  62.194534 T:1399845792  NOTICE: [plugin.video.quasar] Error #01: Unable to retrieve the remote's addon.xml file.
18:47:10  62.380898 T:1399845792  NOTICE: [plugin.video.quasar] [GIN] 2016/12/02 - 16:47:10 |[97;42m 200 [0m|      53.281µs | 127.0.0.1 |[97;45m  [0m HEAD    /repository/scakemyer/plugin.video.quasar/icon.png
18:47:39  91.370544 T:1347416992  NOTICE: [plugin.video.quasar] Requesting http://localhost:65251/trakt/authorize from ['plugin://plugin.video.quasar/trakt/authorize', '-1', '']
18:47:39  91.445862 T:1399845792  NOTICE: [plugin.video.quasar]
18:47:39  91.448982 T:1347416992  NOTICE: Previous line repeats 1 times.
18:47:39  91.449135 T:1347416992   ERROR: [plugin.video.quasar] Traceback (most recent call last):
18:47:39  91.449516 T:1347416992   ERROR: [plugin.video.quasar]   File "/storage/.kodi/addons/plugin.video.quasar/resources/site-packages/quasar/navigation.py", line 112, in run
18:47:39  91.449821 T:1347416992   ERROR: [plugin.video.quasar]     data = _json(url)
18:47:39  91.450302 T:1347416992   ERROR: [plugin.video.quasar]   File "/storage/.kodi/addons/plugin.video.quasar/resources/site-packages/quasar/navigation.py", line 48, in _json
18:47:39  91.450691 T:1347416992   ERROR: [plugin.video.quasar]     with closing(urllib2.urlopen(url)) as response:
18:47:39  91.451004 T:1347416992   ERROR: [plugin.video.quasar]   File "/usr/lib/python2.7/urllib2.py", line 154, in urlopen
18:47:39  91.451424 T:1347416992   ERROR: [plugin.video.quasar]   File "/usr/lib/python2.7/urllib2.py", line 437, in open
18:47:39  91.451767 T:1347416992   ERROR: [plugin.video.quasar]   File "/usr/lib/python2.7/urllib2.py", line 550, in http_response
18:47:39  91.452103 T:1347416992   ERROR: [plugin.video.quasar]   File "/usr/lib/python2.7/urllib2.py", line 475, in error
18:47:39  91.452469 T:1347416992   ERROR: [plugin.video.quasar]   File "/usr/lib/python2.7/urllib2.py", line 409, in _call_chain
18:47:39  91.452766 T:1347416992   ERROR: [plugin.video.quasar]   File "/usr/lib/python2.7/urllib2.py", line 558, in http_error_default
18:47:39  91.453094 T:1347416992   ERROR: [plugin.video.quasar] HTTPError: HTTP Error 500: Internal Server Error
18:47:39  91.453697 T:1399845792  NOTICE: [plugin.video.quasar] [31m2016/12/02 16:47:39 [Recovery] panic recovered:
18:47:39  91.453987 T:1347416992   ERROR: [plugin.video.quasar]
18:47:39  91.584007 T:1399845792  NOTICE: [plugin.video.quasar] GET /trakt/authorize HTTP/1.1
18:47:39  91.588150 T:1399845792  NOTICE: [plugin.video.quasar] Host: localhost:65251
18:47:39  91.588638 T:1399845792  NOTICE: [plugin.video.quasar] Connection: close
18:47:39  91.588943 T:1399845792  NOTICE: [plugin.video.quasar] Accept-Encoding: identity
18:47:39  91.589233 T:1399845792  NOTICE: [plugin.video.quasar] Connection: close
18:47:39  91.589523 T:1399845792  NOTICE: [plugin.video.quasar] User-Agent: Python-urllib/2.7
18:47:39  91.589790 T:1399845792  NOTICE: [plugin.video.quasar]
18:47:39  91.590385 T:1399845792  NOTICE: Previous line repeats 1 times.
18:47:39  91.590424 T:1399845792  NOTICE: [plugin.video.quasar] runtime error: invalid memory address or nil pointer dereference
18:47:39  91.590744 T:1399845792  NOTICE: [plugin.video.quasar] /usr/local/go/src/runtime/panic.go:458 (0x1018b8)
18:47:39  91.591064 T:1399845792  NOTICE: [plugin.video.quasar] /usr/local/go/src/runtime/panic.go:62 (0xffd58)
18:47:39  91.591446 T:1399845792  NOTICE: [plugin.video.quasar] /usr/local/go/src/runtime/sigpanic_unix.go:24 (0x11846c)
18:47:39  91.591805 T:1399845792  NOTICE: [plugin.video.quasar] /go/src/github.com/jmcvetta/napping/request.go:97 (0x51d628)
18:47:39  91.592140 T:1399845792  NOTICE: [plugin.video.quasar] /go/src/github.com/scakemyer/quasar/trakt/trakt.go:358 (0x454510)
18:47:39  91.592484 T:1399845792  NOTICE: [plugin.video.quasar] /go/src/github.com/scakemyer/quasar/trakt/trakt.go:446 (0x45539c)
18:47:39  91.592819 T:1399845792  NOTICE: [plugin.video.quasar] /go/src/github.com/scakemyer/quasar/api/trakt.go:19 (0x214fcc)
18:47:39  91.593155 T:1399845792  NOTICE: [plugin.video.quasar] /go/src/github.com/gin-gonic/gin/context.go:97 (0x338658)
18:47:39  91.593506 T:1399845792  NOTICE: [plugin.video.quasar] /go/src/github.com/gin-gonic/gin/recovery.go:45 (0x347700)
18:47:39  91.593849 T:1399845792  NOTICE: [plugin.video.quasar] /go/src/github.com/gin-gonic/gin/context.go:97 (0x338658)
18:47:39  91.594170 T:1399845792  NOTICE: [plugin.video.quasar] /go/src/github.com/gin-gonic/gin/logger.go:63 (0x346b70)
18:47:39  91.594536 T:1399845792  NOTICE: [plugin.video.quasar] /go/src/github.com/gin-gonic/gin/context.go:97 (0x338658)
18:47:39  91.594856 T:1399845792  NOTICE: [plugin.video.quasar] /go/src/github.com/gin-gonic/gin/gin.go:284 (0x33e138)
18:47:39  91.595177 T:1399845792  NOTICE: [plugin.video.quasar] /go/src/github.com/gin-gonic/gin/gin.go:265 (0x33de44)
18:47:39  91.595505 T:1399845792  NOTICE: [plugin.video.quasar] /usr/local/go/src/net/http/server.go:2022 (0x18fd38)
18:47:39  91.595825 T:1399845792  NOTICE: [plugin.video.quasar] /usr/local/go/src/net/http/server.go:2202 (0x1906bc)
18:47:39  91.596138 T:1399845792  NOTICE: [plugin.video.quasar] /usr/local/go/src/net/http/server.go:1579 (0x18d854)
18:47:39  91.596458 T:1399845792  NOTICE: [plugin.video.quasar] /usr/local/go/src/runtime/asm_arm.s:998 (0x132918)
18:47:39  91.596741 T:1399845792  NOTICE: [plugin.video.quasar] [0m
18:47:39  91.597153 T:1399845792  NOTICE: [plugin.video.quasar] [GIN] 2016/12/02 - 16:47:39 |[97;41m 500 [0m|   68.155319ms | 127.0.0.1 |[97;44m  [0m GET     /trakt/authorize

**Comment by scakemyer** [comment]
There was a cache issue in your first logs which was fixed in scakemyer/quasar@7433173 but nothing about Trakt authorization there. There's a few errors before the recovered panic, which shouldn't happen anyway, and not having the full logs makes it very hard to debug.

**Comment by scakemyer** [comment]
Please open a new issue with full logs if you're still having this problem, it actually looks like a connection issue and if it's the case I'll make sure the daemon handles them better.
